packages/ui-app/src/InputStorage/options/section.ts

```diff
@@ -2,32 +2,21 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { Storage$Sections } from '@polkadot/storage/types';
+import { Section } from '@polkadot/params/types';
+import { Storage$Sections, Storages } from '@polkadot/storage/types';
 import { DropdownOptions } from '../../InputExtrinsic/types';
+import { shouldDisplaySection } from '../../util/shouldDisplaySection';
 
 import map from '@polkadot/storage';
 
-export default function createOptions (): DropdownOptions {
+export default function createOptions (type: string = 'public'): DropdownOptions {
   return Object
     .keys(map)
     .sort()
     .filter((name) => {
-      const section = map[name as Storage$Sections];
+      const section: Section<Storages> = map[name as Storage$Sections];
```

Comment:
Is this type needed here? Type inference will set it up correctly.

Reply:
Agreed, type inference does the trick